app/src/main/java/com/github/se/cyrcle/ui/map/MapScreen.kt

```diff
@@ -938,3 +982,98 @@ fun PreviewCard(navigationActions: NavigationActions, parkingViewModel: ParkingV
         }
   }
 }
+
+/**
+ * Composable function to display the Bike Location Floating action button. Will have different
+ * icons depending on the state of the bikeViewModel.
+ *
+ * @param bikeLocationViewModel The ViewModel for managing bike location-related data and actions.
+ * @param mapViewModel The ViewModel for managing map-related data and actions.
+ * @param navigationActions The actions to navigate to different screens.
+ */
+@Composable
+fun BikeLocationButton(
```

Comment:
5 parameters but 3 documented seems like missing doc to me ^^.

Reply:
Great eyes you have here !